net.puddletag.puddletag.appdata.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>net.puddletag.puddletag</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+  <name>Puddletag</name>
+  <summary>A powerful ID3 tag editor</summary>
+  <description>
+    <p>
+      Puddletag is an audio tag editor. Unlike most taggers for GNU/Linux, it uses a spreadsheet-like layout so that all the tags you want to edit by hand are visible and easily editable.
+    </p>
+    <p>
+      The usual tag editor features are supported like extracting tag information from filenames, renaming files based on their tags by using patterns and basic tag editing.
+    </p>
+    <p>
+      The usual tag editor features are supported:
+      <ul>
+        <li>
+          Extracting tag information from filenames
+        </li>
+        <li>
+          Renaming files based on their tags by using patterns
+        </li>
+        <li>
+          Basic and batch tag editing
+        </li>
+        <li>
+          Automate repetitive tasks
+        </li>
+        <li>
+          Web lookups using Amazon (including cover art), Discogs (does cover art too!), FreeDB and MusicBrainz
+        </li>
+      </ul>
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://docs.puddletag.net/_images/2.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://docs.puddletag.net/_images/5.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://docs.puddletag.net/_images/4.png</image>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://docs.puddletag.net//</url>
+  <update_contact>raduzaharia@outlook.com</update_contact>
```

Comment:
`content_rating` is missing

the file is also never referenced nor installed in the manifest.

Reply:
Ok, added both.